### ISSUE TITLE
gp_segment_configuration shows smdw status

### ISSUE THREAD

**Issue by 0099jerry** [greenplum-db/gpdb#14563]
## Bug Report
We tried to kill Standby session and recovered it. Here are our steps:

1. gpstate -f -> get standby PID
2. kill PID of standby
3. gpstate -f -> Standby process not running 4. 
mxadmin=# select * from gp_segment_configuration order by content,dbid;
 dbid | content | role | preferred_role | mode | status | port | hostname | address |         datadir
------+---------+------+----------------+------+--------+------+----------+---------+-------------------------
    1 |      -1 | p    | p              | n    | u      | 5432 | mdw      | mdw     | /mxdata/master/mxseg-1
    6 |      -1 | m    | m              | s    | u      | 5432 | smdw     | smdw    | /mxdata/standby/mxseg-1
    2 |       0 | p    | p              | s    | u      | 6000 | sdw1     | sdw1    | /mxdata/primary/mxseg0
    4 |       0 | m    | m              | s    | u      | 6001 | sdw2     | sdw2    | /mxdata/mirror/mxseg0
    3 |       1 | p    | p              | s    | u      | 6000 | sdw2     | sdw2    | /mxdata/primary/mxseg1
    5 |       1 | m    | m              | s    | u      | 6001 | sdw1     | sdw1    | /mxdata/mirror/mxseg1
(6 rows)

**Comment by jimmyyih** [comment]
>  We would like to know if it's because FTS will not monitor the status of standby master so the mode and status of standby will not change anyways.

Correct, FTS does not monitor the status of the standby so its gp_segment_configuration entry will not change.  The standby entry in gp_segment_configuration is mainly there to help with gpactivatestandby and provide general information about the standby.

That being said... I did try to make FTS monitor the standby many years back but it was never merged (I think due to an offline discussion where we were undecided on whether or not FTS should monitor the standby). Reference: https://github.com/greenplum-db/gpdb/pull/5177

What are your thoughts/opinions on it?  More user input is always helpful.

**Comment by AManTw** [comment]
Wrong result about standby really makes users confused. :(
When we demo to our parnters, the standby status is wrong.
Can we make this issue a higher priority?

GP version: 6.20.0 
OS version: Ubuntu 18.04.6
